routes/defectReporter.js

```diff
@@ -0,0 +1,60 @@
+const express = require("express");
+const router = express.Router();
+const database = require("../services/database");
+const tables = require("../constants/tableNames");
+const AppError = require("../utils/appError");
+const sendMail = require("../services/sendMail");
+const multer = require('multer');
+const crypto = require('crypto');
+const authentication = require("../middlewares/authentication");
+
+const upload = multer({ 
+    storage: multer.memoryStorage(),
+    limits: { fileSize: 20 * 1024 * 1024, fields: 3 }, // Limit image size to 20MB
+});
+
+let defectReport = {};
+
+router.post('/', authentication, upload.single('image'), async function(req, res, next) {
+    const payload = req.body;
+    const language = payload.language || "de";
+    const userId = req.userId;
+    try {
+        const {title, description } = payload;
+        if (!title || !description || !req.file){
+            return next(new AppError("All fields are mandatory" , 400));
+        }
+
+        const imageHash = crypto.createHash('md5').update(req.file.buffer).digest('hex');
+
+        defectReport = {
+            userId,
+            title,
+            description,
+            hashOfImage: imageHash,
+        };
+        let recipients = JSON.parse(process.env.DEFECT_REPORTING_EMAILS);
```

Comment:
I would suggest you to rather add these to the <db. This is not a secret or a environment specific data. If it is in the DB, it will be easier to maintain

Reply:
In the current requirement, this is a project specific data. So it will likely have only one or two entries. If the requirements change in the future, we can refactor the system and create a dedicated table for this.